public/src/components/channelManagement/testLiveSwitch.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```

Comment:
this file is based on the existing `LiveSwitch` component, which was shared with the amounts tests tool. They're now sufficiently different that I don't think it's worth sharing a component